src/github.com/matrix-org/dendrite/common/consumers.go

```diff
@@ -25,8 +25,14 @@ type PartitionStorer interface {
 // A ContinualConsumer continually consumes logs even across restarts. It requires a PartitionStorer to
 // remember the offset it reached.
 type ContinualConsumer struct {
-	Topic          string
-	Consumer       sarama.Consumer
+	// The kafkaesque topic to consume events from.
```

Comment:
Fields don't have the same linting rules as types or functions, which require them to start with the name. This was directly lifted from Mjark's prose, which I feel adequately describes the purpose of the field. If you prefer, I can change this to start "Topic is...".